ebpf/session.go

```diff
@@ -730,7 +738,7 @@ func (s *session) cleanup() {
 	s.symCache.Cleanup()
 
 	for pid := range s.pids.dead {
-		_ = level.Debug(s.logger).Log("msg", "cleanup dead pid", "pid", pid)
+		//_ = level.Debug(s.logger).Log("msg", "cleanup dead pid", "pid", pid)
```

Comment:
may be we should remove it if we don't need it or uncomment if that is useful debug info.